src/main/java/org/mapfish/print/map/readers/MapReader.java

```diff
@@ -51,5 +53,8 @@ protected boolean canMerge(MapReader other) {
     }
 
     public abstract String toString();
+    public String getPdfLayerName() {
+        return pdfLayerName.isEmpty() || pdfLayerName.isBlank() ? toString() : pdfLayerName;
+    }
```

Comment:
Considering java beans naming convention should result into getPDFLayerName / setPDFLayerName ...

No definitive answer https://stackoverflow.com/questions/2236807/java-naming-convention-with-acronyms
Recommend getPDFLayerName / setPDFLayerName (setter would be used even if just for a test case).

Reply:
Solved!